mon-pix/tests/unit/controllers/campaigns/send-profile-test.js

```diff
@@ -0,0 +1,43 @@
+import { describe, it, beforeEach } from 'mocha';
+import { setupTest } from 'ember-mocha';
+import sinon from 'sinon';
+
+describe('Unit | Controller | Campaigns | Send Profile', function() {
+
+  setupTest();
+
+  const campaignParticipation = {
+    id: 8654,
+    isShared: false,
+    save: sinon.stub(),
+    set: sinon.stub().resolves(),
+  };
+
+  const campaignParticipationShared = { ...campaignParticipation, isShared: true };
+
+  const model = {
+    campaign: {
+      id: 1243,
+      code: 'CODECAMPAIGN',
+    },
+    campaignParticipation
+  };
+  let controller;
+
+  beforeEach(function() {
+    controller = this.owner.lookup('controller:campaigns/send-profile');
+    controller.set('model', model);
+    campaignParticipation.save.resolves(campaignParticipationShared);
+  });
+
+  describe('#sendProfile', function() {
```

Comment:
Il manque des tests unit pour le reject et les  `this.set('errorMessage', null)` et `this.set('isLoading', true)`